### PR TITLE
feature/widget-board

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
#### Summary
- The list of widgets on the assets page;
- Board is visible only if the query string contains `feature=widgets`.

#### Screenshots
image
TotalMarketcap Widget for the watchlist:
image
Stacking Total marketcap and top 3 watchlist projects (on the picture there are only 2 projects)
image

### CODE REVIEW

**Comment by Partysun** [comment]
Did you use special API projectListHistoryStats, instead of a lot of queries from a client side? @DmitriVanGuard

File: app/src/components/TotalMarketcapWidget/TotalMarketcapGQL.js [hunk 1]
```diff
@@ -9,3 +9,45 @@ export const totalMarketcapGQL = gql`
...
+  query projectsListHistoryStats(
+    $from: DateTime!
+    $to: DateTime!
+    $slugs: [String]!
+  ) {
+    projectsListHistoryStats(
+      from: $from
+      to: $to
+      slugs: $slugs
+      interval: "1d"
```
Comment (Partysun) [inline app/src/components/TotalMarketcapWidget/TotalMarketcapGQL.js]: @tspenov Can we make a default value in the backend side? for interval. We have defaulted in other same methods.

Comment (tspenov) [reply app/src/components/TotalMarketcapWidget/TotalMarketcapGQL.js]: @Partysun will change

Comment (tspenov) [reply app/src/components/TotalMarketcapWidget/TotalMarketcapGQL.js]: @Partysun @DmitriVanGuard there is `1d` default value now

Comment (DmitriVanGuard) [reply app/src/components/TotalMarketcapWidget/TotalMarketcapGQL.js]: @tspenov thank you!

**Pull request merged**